lib/tasks/deadline_reminder.rake

```diff
@@ -0,0 +1,15 @@
+namespace :deadline_reminder do
+
+  desc "send email reminder to mentors and mentees who did not decide on the project after first week"
+  task project_setup_reminder: :environment do
+    mentees_email_template = EmailTemplate.find_by(recipients: 15)
+    mentors_email_template = EmailTemplate.find_by(recipients: 16)
+    mentees_email_template.users.each do |mentee|
+      EmailTemplateMailer.custom(mentees_email_template, mentee).deliver_now
```

Comment:
@DMips It was working fine for me. At what point do you think it would be failing due to registration token ?